pyflakes/checker.py

```diff
@@ -887,6 +909,7 @@ def handleNode(self, node, parent):
         self.nodeDepth += 1
         node.depth = self.nodeDepth
         node.parent = parent
+        self.handleTypeComments(node)
```

Comment:
This didn't feel right as well, but worked. Again happy for improvement suggestions.